src/Common/src/Interop/Unix/System.Native/Interop.Poll.cs

```diff
@@ -20,9 +21,10 @@ internal enum PollEvents : short
 
         internal struct PollEvent
         {
-            internal int FileDescriptor;         // The file descriptor to poll
+            internal IntPtr FileDescriptor;      // The file descriptor to poll
```

Comment:
I'm going to revert this change, and along with that not change `Sys.Pipe` to take in IntPtr[].  Both of these changes come at a cost, with minimal benefits.
@stephentoub @nguerrera - Are you guys OK with this?